pkg/handler/nfts.go

```diff
@@ -28,6 +28,19 @@ func (h *Handler) CreateNFTCollection(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
 		return
 	}
+
+	checkExitsNFT, err := h.entities.CheckExistNftCollection(req.Address)
```

Comment:
move this logic behind `checksumAddress, _ := util.ConvertToChecksumAddr(req.Address)`
use the checksumAddress instead of req.Address